runtime/ftplugin/man.vim

```diff
@@ -31,7 +31,7 @@ if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
   nnoremap <silent> <buffer> j             gj
   nnoremap <silent> <buffer> k             gk
   nnoremap <silent> <buffer> gO            :call man#show_toc()<CR>
-  nnoremap <silent> <buffer> <2-LeftMouse> :Map<CR>
+  nnoremap <silent> <buffer> <2-LeftMouse> :Man<CR>
```

Comment:
Let's squash this into the previous commit.

Reply:
Done!